binding/ios/Orca.swift

```diff
@@ -56,6 +237,24 @@ public class Orca {
         }
     }
 
+    /// Maximum number of characters allowed per call to `synthesize()`.
+    public var maxCharacterLimit: Int32 {
+        get throws {
+            if _maxCharacterLimit == nil {
+                var cMaxCharacterLimit: Int32 = 0
+                let status = pv_orca_max_character_limit(handle, &cMaxCharacterLimit)
+                if status != PV_STATUS_SUCCESS {
+                    let messageStack = try getMessageStack()
+                    throw pvStatusToOrcaError(status, "Orca failed to get max character limit", messageStack)
+                }
+
+                _maxCharacterLimit = cMaxCharacterLimit
+            }
+
+            return _maxCharacterLimit!
+        }
+    }
```

Comment:
I much prefer that our post-init constants are fetched on initialization rather than getting them at runtime every time. If you just set `maxSampleRate`, `sampleRate` and `validCharacters` to swift constants during the call to `init()` then the used doesn't have to wrap getting these values in a try/catch. Same goes for Android actually. See how in Rhino iOS we get contextInfo in the [init](https://github.com/Picovoice/rhino/blob/master/binding/ios/Rhino.swift#L129). Improves the stability of the user app and means less code the user has to write to use our library.